docs/sdk/php/README.mdx

```diff
@@ -0,0 +1,81 @@
+---
+sidebar_label: PHP
+---
+
+import ApiResourcesDescription from '../fragments/_api_resources_description.md';
+import FurtherReadings from '../fragments/_further-readings.md';
+import ImplementCallbackRoute from './_implement_callback_route.md';
+import ImplementHomePage from './_implement_home_page.md';
+import ImplementSignInRoute from './_implement_sign_in_route.mdx';
+import ImplementSignOutRoute from './_implement_sign_out_route.mdx';
+import InitLogtoClient from './_init_logto_client.mdx';
+import Installation from './_installation.md';
+import PhpTutorialTip from './_php_tutorial_tip.md';
+import Prerequisites from './_prerequisites.md';
+import TestYourApplication from './_test_your_application.md';
+import ConfigApiResources from './api-resources/_config_api_resources.mdx';
+import FetchAccessTokenForApiResources from './api-resources/_fetch_access_token_for_api_resources.mdx';
+import FetchOrganizationTokenForUser from './api-resources/_fetch_organization_token_for_user.mdx';
+import ScopesAndClaims from './scopes-and-claims/_scopes_and_claims.mdx';
+
+# PHP SDK tutorial
+
+<PhpTutorialTip />
+
+## Prerequisites
+
+<Prerequisites />
+
+## Installation
+
+<Installation />
+
+## Integration
+
+### Init LogtoClient
+
+<InitLogtoClient />
+
+### Implement sign-in route
+
+<ImplementSignInRoute />
+
+### Implement the callback route
+
+<ImplementCallbackRoute />
+
+### Implement the home page
```

Comment:
In other guides, the "home" and "callback" section is the sub-section of "Implement sign-in".

Reply:
Updated.